js/projects.js

```diff
@@ -0,0 +1,169 @@
+      'CSS',
+      'Javascript',
+    ],
+  },
+];
+
+project.innerHTML = projectData.map((project) => (
+  `<div class="second-project project"> 
+      <h4 class="project-title">${project.name}</h4>
+      <p class="project-desc">
+      ${project.description}
+      </p>
+      <ul class="project-techs">
+        ${project.languages.map((lang) => `<li>${lang}</li>`).join('')}
+      </ul>
+      <button type="submit" class="check-project">
+      See Project
+    </button>
+    </div>
+    `
+)).join('');
+
+const modalCardData = [
+  {
+    name: 'Data Dashboard Healthcare ',
+    image: '../images/Snapshoot Portfolio.png',
+    description: `Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quasi assumenda temporibus autem voluptates, nemo vero culpa optio quae? Tempora aliquam maxime tenetur iusto vero minima similique illum recusandae est deleniti!
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab sunt corrupti cupiditate quaerat quod assumenda tempore, eius temporibus officia error asperiores quasi! Tempora iusto quaerat nam voluptatibus, facere iure exercitationem.
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis harum laboriosam saepe? Placeat culpa quo dolorum soluta. Beatae, blanditiis saepe aliquam id alias iusto rem voluptatibus, provident doloremque hic odit?`,
+    languages: [
+      'HTML',
+      'Bootstrap',
+      'Ruby on Rails',
+    ],
+    liveUrl: 'https://efiamotu-1.github.io/Portfolio-Mobile-Version-skeleton/',
+    sourceUrl: 'https://github.com/Efiamotu-1/Portfolio-Mobile-Version-skeleton',
+
+  },
+];
+
+modal.innerHTML = modalCardData.map((modal) => (
+  `
+    <div class="modal" id="modal">
+    <div class="modal-header">
+      <h2 id="modal-title">${modal.name} </h2>
+      <span class="modal-close" id ="modal-close">&times;</span>
+    </div>
+    <div id="modal-languages">
+      <ul class="works-categories">
+        ${modal.languages.map((langSkill) => `<li class="work-skills">${langSkill}</li>`).join('')}     
+        </ul>
+    </div>
+    <div class="modal-content">
+      <div class="featured-image">
+        <img id="project-image" src="${modal.image}" alt="Project Preview">
+      </div>
+      <div class="content">
+        <p id="modal-description"> 
+         ${modal.description}
+        </p>
+        <div id="link-container" class="link-container">
+          <a id="live-link" class="button live-icon" href="${modal.liveUrl}" target="new">See Live</a>
+          <a id="source-link" class="button source-icon" href="${modal.sourceUrl}" target="new">See Source</a>
+        </div>
+      </div>
+    </div>
+  </div>
+    `
+)).join('');
+const openModal = document.querySelectorAll('.button, .check-project');
+const container = document.querySelector('.container-element');
+openModal.forEach((open) => {
+  open.addEventListener('click', () => {
+    modalContainer.style.display = 'block';
+    container.style.filter = 'blur(5px)';
+  });
+});
```

Comment:
- [ ] Kindly be noticed that the projects section is created dynamically with the information from the projects array.
- [ ] Also notice modal popup data is populated with the data from the projects array, so when the user clicks on a show project button it should be this project to show not fixed data.